clang/test/CheckedCRewriter/arrbothmulti2.c

```diff
@@ -129,5 +129,8 @@ z += 2;
 return z; }
 //CHECK_NOALL: int * sus(int *x, _Ptr<int> y) {
 //CHECK_NOALL:         int *z = calloc(5, sizeof(int)); 
-//CHECK_ALL: int * sus(int *x, _Ptr<int> y) {
-//CHECK_ALL:         int *z = calloc(5, sizeof(int)); 
+//CHECK_ALL: _Nt_array_ptr<int> sus(int *x, _Ptr<int> y) {
+//CHECK_ALL:         _Nt_array_ptr<int> z =  calloc(5, sizeof(int)); 
+//CHECK_ALL:         forint i = 0;
```

Comment:
Same question as above ... Not sure what the other lines are below, either. Should `_Array_ptr<int> p = z;` etc. be deleted?